osf_pigeon/app.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import requests
+from osf_pigeon import pigeon
+from concurrent.futures import ThreadPoolExecutor
+from osf_pigeon import settings
+from aiohttp import web
+
+import sentry_sdk
+from sentry_sdk.integrations.aiohttp import AioHttpIntegration
+
+sentry_sdk.init(
+    dsn=settings.SENTRY_DSN,
+    release="0.0.10",
+    integrations=[AioHttpIntegration()],
+)
+
+pigeon_jobs = ThreadPoolExecutor(max_workers=1, thread_name_prefix="pigeon_jobs")
+app = web.Application()
+routes = web.RouteTableDef()
+logging.basicConfig(filename="pigeon.log", level=logging.DEBUG)
+
+
+def handle_exception(future):
+    exception = future.exception()
+    if exception:
+        sentry_sdk.capture_exception(exception)
+        app.logger.exception(exception)
+
+
+def archive_task_done(future):
+    if future._result and not future._exception:
+        ia_item, guid = future.result()
+        resp = requests.post(
+            f"{settings.OSF_API_URL}_/ia/{guid}/done/",
+            json={"ia_url": ia_item.urls.details},
+        )
+        app.logger.info(f"{ia_item} called back with {resp}")
+
+
+def metadata_task_done(future):
+    if future._result and not future._exception:
+        ia_item, updated_metadata = future.result()
+        app.logger.info(f"{ia_item} updated metadata {updated_metadata}")
+
+
+@routes.get("/")
+async def index(request):
+    return web.json_response({"🐦": "👍"})
+
+
+@routes.get("/logs")
+async def logs(request):
```

Comment:
I suspect this may need to be removed (or edited so it's only accessible within the VPN?)...Will get input from MattF

Reply:
Yeah it should be protected by the network policy, but it's not great practice and should be removed once sentry can report errors itself.